src/sentry/runner/initializer.py

```diff
@@ -314,6 +314,9 @@ def initialize_app(config, skip_service_validation=False):
         settings.CSRF_COOKIE_PATH = getattr(settings, "SESSION_COOKIE_PATH", "/")
 
     settings.CACHES["default"]["VERSION"] = settings.CACHE_VERSION
+    for key in settings.CACHES:
+        if not hasattr(settings.CACHES[key], "KEY_PREFIX"):
+            settings.CACHES[key]["KEY_PREFIX"] = "v0:" if six.PY3 else ""
```

Comment:
Why don't we do this:

```python
if six.PY3:
    for key in settings.CACHES:
        settings.CACHES[key]["VERSION"] = 2
```

There are 2 options relevant here, the KEY_PREFIX, and the VERSION. We really only wanna bump the version then we don't make the key names longer (which use more memory). The version defaults to 1 already, so we just bump this value instead.

https://github.com/django/django/blob/stable/1.11.x/django/core/cache/backends/base.py#L27-L35